source/code-generation/query-converter/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,63 @@
+.. _rm-ai-and-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The Query Converter feature is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+Query Converter currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
+How Your Data is Used
+---------------------
+
+When you use Query Converter, the following information is sent to 
+MongoDB's backend and the third party AI provider:
+
+- The full text of your query or stored procedure.
+- The definition of any relational database tables used in the query.
+  These definitions include the table names, column names, data types,
+  and any primary or foreign keys relationships.
+- The definition of your MongoDB collections and mapping rules. 
+  These definitions include the relationships between the collections 
+  and tables used in the query and the collection names, field names 
+  and data types.
+
+The information that is sent is not shared with any other third 
+parties or stored by the AI provider. Relational Migrator does not send 
+database connection strings, credentials, or any raw data from your 
+databases.
+
+By default, your original query text and the AI-converted query 
+telemetry data is stored by MongoDB for up to one year to help us 
+provide support and improve the service. If you do not want
+your data to be stored, you have the option to opt out:
+
+- You can disable the :guilabel:`Allow queries to be stored temporarily` 
+  toggle switch in the global project settings.
+- You can uncheck the :guilabel:`Allow MongoDB to temporarily store the 
+  source and converted queries to import the product feature` setting 
+  when you enable the query converter.
+
+Disable Query Converter
+-----------------------
+
```

Comment:
[Question]

Do you think we could link to the Enable Query Converter page here?